contrib/runners/mistral_v2/query/mistral_v2.py

```diff
@@ -55,24 +61,43 @@ def __init__(self, id, *args, **kwargs):
         wait_exponential_multiplier=cfg.CONF.mistral.retry_exp_msec,
         wait_exponential_max=cfg.CONF.mistral.retry_exp_max_msec,
         stop_max_delay=cfg.CONF.mistral.retry_stop_max_msec)
-    def query(self, execution_id, query_context):
+    def query(self, execution_id, query_context, last_query_time=None):
         """
         Queries mistral for workflow results using v2 APIs.
         :param execution_id: st2 execution_id (context to be used for logging/audit)
         :type execution_id: ``str``
         :param query_context: context for the query to be made to mistral. This contains mistral
                               execution id.
-        :type query_context: ``objext``
+        :type query_context: ``object``
+        :param last_query_time: Timestamp of last query.
+        :type last_query_time: ``float``
         :rtype: (``str``, ``object``)
         """
+        dt_last_query_time = (
+            time.strftime('%Y-%m-%d %H:%M:%S', time.gmtime(last_query_time))
```

Comment:
https://docs.python.org/2/library/time.html#time.gmtime says time.gmtime returns UTC and daylight savings time not represented. i assume it takes the epoch seconds and just return the UTC, doesn't care about DST. epoch seconds is already in UTC per https://en.wikipedia.org/wiki/Unix_time#Encoding_time_as_a_number. and since mistral already saves timestamp as UTC. so afaik, DST doesn't apply here...